README.md

```diff
@@ -1 +1,170 @@
-# spello
\ No newline at end of file
+<p align="center">
+    <img src="logo.png" width="350", height="220">
+</p> 
+<p align="center">A Fast and Smart SpellCorrection using Sound and Edit-distance based Correction available in English and 10 indian languages.      
+</p>  
+<p align="center">  
+  <a href="https://github.com/hellohaptik/spello/stargazers">  
+    <img src="https://img.shields.io/github/stars/hellohaptik/spello.svg?colorA=orange&colorB=orange&logo=github"  
+         alt="GitHub stars">  
+  </a> 
+  <a href="https://pepy.tech/project/spello/">  
+      <img src="https://pepy.tech/badge/spello" alt="Downloads">  
+  </a>   
+  <a href="https://pypi.org/project/spello/">  
+      <img src="https://img.shields.io/pypi/v/spello?colorB=brightgreen" alt="Pypi package">  
+  </a>  
+  <a href="https://github.com/hellohaptik/spello/issues">
+        <img src="https://img.shields.io/github/issues/hellohaptik/spello.svg"
+             alt="GitHub issues">
+  </a>
+  <a href="https://github.com/hellohaptik/spello/blob/master/LICENSE">  
+        <img src="https://img.shields.io/github/license/hellohaptik/spello.svg"  
+             alt="GitHub license">  
+  </a>
+  <a href="https://github.com/hellohaptik/spello/graphs/contributors">  
+        <img src="https://img.shields.io/badge/all_contributors-5-blue.svg"  
+             alt="Contributors">  
+  </a>  
+</p>  
+  
+<p align="center">  
+ <a href="#what-is-it">What is it</a> •  
+  <a href="#-installation">Installation</a> •  
+  <a href="#-️getting-started">Getting Started</a> 
+</p>  
+</p>
+
+<h2 align="center">What is it</h3>  
+  
+**Spello** is a spellcorrection model using power of two model Symspell and Phoneme in backend to get the best possible spelling suggestion for misspelled words in a text. Phoneme Model uses <a href="https://en.wikipedia.org/wiki/Soundex">Soundex</a> algo in background and is responsible to suggest corrections for sound related mistakes in word. Also, we are using a modifield version of<a href="https://github.com/wolfgarbe/SymSpell"> Symspell</a> model to get spell suggestions based on edit-distances. <br>
+Currently, this module is available for **English(en)** and 10 other indian languages which are **Hindi(hi)**,  **Marathi(mr)**,  **Bengali(bn)**,  **Punjabi(pa)**, **Gujarati(gu)**, **Oriya(or)**, **Tamil(ta)**, **Telegu(tl)**, **Kannada(kn)**, **Malyalam(ml)**.  
```

Comment:
We have not tested it for other languages and we don't know how it works. So we just say english and Hindi. We are not giving pretrained models for other languages also.

Reply:
Sure